tests/github-action-helper.sh

```diff
@@ -267,6 +267,27 @@ install_minikube_with_none_driver() {
   sudo -E minikube start --kubernetes-version="$1" --driver=none --memory 6g --cpus=2 --addons ingress --cni=calico
 }
 
+install_external_snapshotter() {
+  curl -L https://github.com/kubernetes-csi/external-snapshotter/archive/refs/tags/v6.2.0.zip -o external-snapshotter.zip
+  unzip external-snapshotter.zip
+  cd external-snapshotter-6.2.0
+
+  kubectl kustomize client/config/crd | kubectl create -f -
+  kubectl -n kube-system kustomize deploy/kubernetes/snapshot-controller | kubectl create -f -
+}
+
+wait_for_rbd_pvc_clone_to_be_bound() {
+  kubectl create -f https://raw.githubusercontent.com/rook/rook/master/deploy/examples/csi/rbd/pvc-clone.yaml
```

Comment:
i think you are referring to pvc-pvc clone here not pvc-restore right?

where are we creating the volumesnapshotclass and volumesnapshot, if its planned in different PR can we make the changes related to it in same PR instead of here?

Reply:
I'll add other tests in another PR as you suggested. 

https://github.com/rook/kubectl-rook-ceph/pull/229#issuecomment-2084428627
> Lets not block this PR for it but good to have tests to cover all cases, we can open a new issue to add more tests to ensure it works for all the cases

 I'll create an issue to track this test improvement.